lib/puppet/module/task.rb

```diff
@@ -61,6 +62,25 @@ def self.is_tasks_filename?(path)
       return true
     end
 
+    def self.find_files(name, metadata)
+      if metadata.key?('files')
+        file_list = metadata['files'].map do |file|
+          splits = file.split(File::SEPARATOR)
+          pup_module = Puppet::Module.find(splits[0])
```

Comment:
What happens when the module isn't found?

Reply:
Do we care if the file isn't found in the module at this point? There may be some weirdness since we lookup files outside of jruby in puppet-server.